### PR TITLE
Add test on CliRouter to check cleanup

### PR DESCRIPTION
**Pull request by ghost**
Test to check `CliRouter` removes std-in file on its destructor correctly.
This make sure fix #227 still works.

### CODE REVIEW

File: tests/Provide/Router/CliRouterTest.php [hunk 1]
```diff
@@ -100,4 +100,14 @@ public function testError()
         ];
         $this->router->match($globals, []);
     }
+
+    public function testStdInCleanup()
+    {
+        file_put_contents($this->stdInFile, '');
+        $exists = file_exists($this->stdInFile);
+        $this->assertTrue($exists);
+        unset($this->router);
```
Comment (ghost) [inline tests/Provide/Router/CliRouterTest.php:109]: I'm not sure the best way to test destructors.
I used `unset` to dereference. Can we call `__destruct` explicitly?

Comment (koriym) [reply tests/Provide/Router/CliRouterTest.php:109]: should be no problem.

Comment (koriym) [reply tests/Provide/Router/CliRouterTest.php:109]: or you can `unset` ?

Comment (koriym) [reply tests/Provide/Router/CliRouterTest.php:109]: oh you did it already.  good.

Comment (ghost) [reply tests/Provide/Router/CliRouterTest.php:109]: @koriym 
Thank you for the advice! I see.
Both case works, so I'll go with this (`unset`) code.

Comment (koriym) [reply tests/Provide/Router/CliRouterTest.php:109]: awesome !

**Pull request merged**